chapters/use-deliver/overview/reading/read.md

```diff
@@ -1,22 +1,38 @@
 # Using and Delivering Content
 
-Owner: Liza Babu
-Reviewer: Ștefan Jumărea
+With content being developed and available in the content repositories, educators can configure it, deploy it and use it for their educational activities.
+Content can also be used by learners during self-study activities.
+When educators use content, they both deliver it live, during educational activities, and make it available publicly for self-study activities.
 
-Present:
+Educators fork the content repository into a delivery repository.
+They configure the content according to their own needs, preferences or specifics of the learner group.
+They then use the deployment and publishing infrastructure to make the content available to learners, both during and outside educational activities.
+Optionally, educators may also fork solution repositories into a solution delivery repositories.
 
-- who the implementation specific repository is organized, by referring content in the content repository
-- how components of the actual education delivery phase are used;
-  each is having a dedicated section
-
-This chapter describes how each teaching activity is mapped to different types of content.
-
-The mapping between teaching activities and types of content is presented in the [Introduction](../../../landing-page/read.md).
-
-Sections:
+Educators organize content for the different educational activities they employ.
+The mapping between educational activities and types of content is presented in the [Introduction](../../../landing-page/read.md).
+The topics presented in this chapter are dedicated to each educational activity:
 
 - [Lectures](../../lectures/reading/read.md)
 - [Practical Sessions](../../practical-sessions/reading/read.md)
 - [Assignments](../../assignments/reading/read.md)
 - [Tests](../../tests/reading/read.md)
 - [Self-Study](../../self-study/reading/read.md)
+
+In the ideal case, an educator is only required to update the `config.yaml` file in the delivery repository.
+The rest of the content is left unaltered and is thus in perfect sync with the original content repository.
+
+If changes to content are required, the educator is recommended to push the changes to the "upstream" content repository.
```

Comment:
```suggestion
If changes to content are required, the educator is recommended to push the changes to the "upstream" content repository (the one under the Open Education Hub organization).
```
It may not be clear to someone not using git what "upstream" means.